index.html

```diff
@@ -204,20 +220,24 @@ <h1 class="card-title">Profesional Art Printing Data</h1>
     </section>
     <section class="about">
       <div class="about-div">
-        <h1 class="about-heading">About me</h1>
-        <p class="about-descrp">
-          Hello I’m a software developer! I can help you build a product ,
-          feature or website Look through some of my work and experience! If you
-          like what you see and have a project you need coded, don’t hestiate to
-          contact me.
-        </p>
-        <button class="about-btn">Get My Resume</button>
+        <div class="contents-about">
+          <h1 class="about-heading">About me</h1>
+          <p class="about-descrp">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <button class="about-btn">Get My Resume</button>
+        </div>
       </div>
       <div class="languages">
```

Comment:
- [ ] [REQURED] Let's ensure the languages are properly aligned as in the figma template as well as it is a core requirement.
### Your Design
![Screenshot from 2022-06-09 12-52-39](https://user-images.githubusercontent.com/58692590/172847128-c591f9ae-3630-4b36-85c6-6c251d631435.png)
### Figma Template 2
![Screenshot from 2022-06-09 13-30-44](https://user-images.githubusercontent.com/58692590/172847410-39023092-ca77-44bd-b6d2-d05b7c382cdd.png)